build/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+#Maintainer: Karl Wikström <boogrocha@sidus.io>
+#Maintainer: William Leven <boogrocha@sidus.io>
+
+_tag=
+pkgname=boogrocha
+pkgver=$_tag
+pkgrel=1
+pkgdesc="A lightweight, easy to use command-line application (bgc) for managing your group room bookings at Chalmers University of Technology"
+arch=('x86_64')
+conflicts=(bgc)
+provides=(bgc)
+url="https://github.com/sidusio/boogrocha"
+license=("GPL3")
+makedepends=(go)
+source=($url/archive/$_tag.tar.gz)
+sha256sums=
+# The name of the root directory of the tar.gz that Github
+# automatically creates and which therefore will be extracted
+_src="BooGroCha-${pkgver:1}"
+
+build() {
+    cd $_src
+    # Build as per Go package guidelines
+    go build \
```

Comment:
what does all this stuff do?

Reply:
https://wiki.archlinux.org/index.php/Go_package_guidelines#Flags_and_build_options